Marlin/src/lcd/tft/tft.h

```diff
@@ -30,6 +30,12 @@
 
 #include "../../inc/MarlinConfig.h"
 
+#if IS_ANET_ET
+  #define ENDIAN_COLOR(C) (((C) >> 8) | ((C) << 8))
```

Comment:
This ENDIAN_COLOR is needed by any 8 bit interface, in the TFT code. It's not a ANET thing.